staging/src/k8s.io/apiserver/pkg/apis/config/v1/types.go

```diff
@@ -86,6 +86,9 @@ type IdentityConfiguration struct{}
 
 // KMSConfiguration contains the name, cache size and path to configuration file for a KMS based envelope transformer.
 type KMSConfiguration struct {
+	// apiVersion of KeyManagementService
+	// +optional
+	APIVersion string `json:"apiVersion"`
```

Comment:
I would expect this to default to "v1" in staging/src/k8s.io/apiserver/pkg/apis/config/v1/defaults.go

Reply:
Thanks for the pointer! Added it.